### ISSUE TITLE
Keyboard remapping using `setxkbmap` not working for external keyboards

### ISSUE THREAD

**Issue by matemat13** [klaxalk/linux-setup#62]
When you connect an external keyboard after the `.i3/config` file was already processed, the keyboard remappings using `setxkbmap` (mapping Esc to CapsLock, switching keyboard locale) are not applied to it. This can be fixed by rerunning the remapping commands manually.

### Possible fix:
Using udev rules to detect when a new keyboard is plugged in and run a script to reapply the remappings (see https://unix.stackexchange.com/questions/28548/how-to-run-custom-scripts-upon-usb-device-plug-in).

**Comment by matemat13** [comment]
### Better fix:
Add the keyboard settings to `/etc/default/keyboard`:
```
# KEYBOARD CONFIGURATION FILE

# Consult the keyboard(5) manual page.

XKBMODEL="pc101"
XKBLAYOUT="us,cz"
XKBVARIANT=",qwerty"
XKBOPTIONS="grp:rctrl_toggle,caps:escape"

BACKSPACE="guess"
```

This way these settings are consistently applied to all keyboards without any hacky ways with bash scripts etc.

### TODO:
Add this setting to linux-setup install so that it is applied when the install script is run.

**Comment by matemat13** [comment]
Should be fixed in commit 66a066b.
